src/commands/buildCommands/trn.js

```diff
@@ -36,10 +36,17 @@ const writeTrnModules = messagesByLocale => ({ filename, msgids }) => {
 		return acc;
 	}, {});
 
-	if (packageConfig.combineLanguages) {
+	if (
```

Comment:
It would be nice if this isn't only in package json. I'm assuming by default we don't want this to be the new default `package.json` configuration in mup-web as this would slow down dev build time.

Reply:
This is a way for apps to opt-in to having a combined bundle. In general, app-specific config is supplied by `package.json` - is there another place that would be useful?

for `build trn`, timing doesn't change noticeably whether you're writing a combined trn source or language-specific source.

Reply:
🤔 build times have definitely felt much longer when i tack on other languages

Reply:
Thats why i was suggesting its another build cli option vs package.json driven at least for mup-web

Reply:
this is a different setup than what we've had before - it will build _one_ bundle supporting _all_ languages, rather than one bundle for each language. It is currently slow to build more than one language because the whole app has to be re-bundled for each language. With this setup, the _only_ cost to supporting more languages is that the application script/bundle is larger, which isn't a concern in dev

Reply:
Okey dokey